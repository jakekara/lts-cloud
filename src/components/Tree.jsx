import React from 'react';
import Diagram from './Diagram';

const Tree = (props) => (
  <div className='tree'>
    <div className='left'>
      <div className='text'>
        <h2>Anthology</h2>
        <h1>Select a word to explore stories of survivors</h1>
      </div>
    </div>
    <div className='right'>
      <Diagram />
    </div>
  </div>
)

export default Tree;