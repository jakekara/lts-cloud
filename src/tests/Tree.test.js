import React from 'react';
import Tree from '../components/Tree';
import { createBrowserHistory } from 'history'
import { Router } from 'react-router';
import renderer from 'react-test-renderer';

it('renders', () => {
  const tree = renderer
    .create(
      <Router history={createBrowserHistory()}>
        <Tree/>
      </Router>
    ).toJSON();
  expect(tree).toMatchSnapshot();
});