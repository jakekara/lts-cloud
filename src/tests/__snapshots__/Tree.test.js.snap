// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="tree"
>
  <div
    className="left"
  >
    <div
      className="text"
    >
      <h2>
        Anthology
      </h2>
      <h1>
        Select a word to explore stories of survivors
      </h1>
    </div>
  </div>
  <div
    className="right"
  >
    <div
      className="diagram"
    >
      <div
        id="hidden"
      />
      <div
        id="target"
      />
    </div>
  </div>
</div>
`;
